content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -16,10 +16,6 @@ content_type: task
 -->
 
 <!-- overview -->
-
-<!--
-{{< glossary_definition term_id="etcd" length="all" prepend="etcd is a ">}}
--->
 {{< glossary_definition term_id="etcd" length="all" prepend="etcd 是一个">}}
```

Comment:
Based on the preview: https://deploy-preview-23979--kubernetes-io-master-staging.netlify.app/zh/docs/tasks/administer-cluster/configure-upgrade-etcd/

The prepend can be deleted.

Reply:
done